packages/tradershub/src/hooks/useCurrencies.ts

```diff
@@ -80,11 +80,28 @@ const useCurrencies = () => {
         [currencyConfig?.CRYPTO]
     );
 
+    // Get the current account currency with its config and isAdded status
+    const currentAccountCurrency = useMemo(() => {
```

Comment:
```suggestion
    const currentAccountCurrencyConfig = useMemo(() => {
```

Reply:
this is not the config but the current currency for the account with the information. The config we're generating above is named `currencyConfig` which will kinda be similar as `current` as it is being generated from current account

Reply:
As we discussed this function will return configs from `website_status?.currencies_config` for a given currency and in addition to `id` , `loading` and some other stuff, so better to rename it to `currentAccountCurrencyConfig`.
<img width="387" alt="image" src="https://github.com/binary-com/deriv-app/assets/82078941/1e42cc70-c56a-482a-9e05-16cba8f67224">

Reply:
Updated @mahdiyeh-deriv